src/Eccube/Resource/template/default/Shopping/shipping_edit.twig

```diff
@@ -22,6 +22,13 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 {% extends 'default_frame.twig' %}
 
 {% set body_class = 'cart_page' %}
+{#ログイン状態によって、表示内容を切替#}
```

Comment:
Use `title` & `subtitle` for twig template will automatically append to `default_template.twig`
[Here](https://github.com/EC-CUBE/ec-cube/blob/master/src/Eccube/Resource/template/default/default_frame.twig#L27)

It also teaches customers who will customize the template to know how to change the title of the twig template only.